tests/Unit/ViewModels/TransactionViewModelTest.php

```diff
@@ -312,6 +312,33 @@
 
         expect($transaction->formattedPayload())->toBe('MethodID: 0x12341234');
     });
+
+    it('should get formatted multi payment receipts', function () {
+        $transaction = new TransactionViewModel(Transaction::factory()
+            ->withPayload('084ce708000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000002000000000000000000000000b693449adda7efc015d87944eae8b7c37eb1690a000000000000000000000000b693449adda7efc015d87944eae8b7c37eb1690a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000005f5e100000000000000000000000000000000000000000000000000000000000bebc200')
```

Comment:
_should_ be able to do the below:

```suggestion
            ->multiPayment([
                '0xb693449AdDa7EFc015D87944EAE8b7C37EB1690A',
                '0xb693449AdDa7EFc015D87944EAE8b7C37EB1690A'
            ], [
                BigNumber::new(100000000), 
                BigNumber::new(100000000),
            ])
```

Reply:
@alexbarnsley that factory does not produce the same payload so should be wrong. I am going to update the factory 🤔